subprojects/snapshots/src/test/groovy/org/gradle/internal/snapshot/CaseInsensitivePathUtilTest.groovy

```diff
@@ -29,8 +29,8 @@ class CaseInsensitivePathUtilTest extends AbstractCasePathUtilTest {
         char char1 = left as char
         char char2 = right as char
         expect:
-        equalChars(char2, char1, false)
-        equalChars(char1, char2, false)
+        equalChars(char2, char1, CaseSensitivity.CASE_INSENSITIVE)
+        equalChars(char1, char2, CaseSensitivity.CASE_INSENSITIVE)
```

Comment:
Maybe use static import for this? I think it would make sense to do it probably everywhere.

Reply:
I added the static import.